run-checks.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import mypy.api
+import pytest
+import pylint.lint
+import sys
+
+import doctests
+import fuzzing
+
+parser = argparse.ArgumentParser(
+    description="Run all or some checks on the codebase. Stops at first failed check"
+)
+
+parser.add_argument(
+    "--unit-tests", "-u",
+    action="store_true",
+    help="Run pytest on tests/"
+)
+
+parser.add_argument(
+    "--doc-tests", "-d",
+    action="store_true",
+    help="Run tests from examples in Readme.md"
+)
+
+parser.add_argument(
+    "--fuzzing", "-f",
+    action="store_true",
+    help="Run fuzzing on parsing logic"
+)
+
+parser.add_argument(
+    "--lint", "-l",
```

Comment:
`--lint` or `--linting` ?

Reply:
`--lint` would be consistent with the other non-conjugated verbs.  I think `--fuzzing` should therefore be `--fuzz` too.